src/app/events/components/search/search.component.html

```diff
@@ -3,8 +3,36 @@
   <ion-searchbar placeholder="Търси събитие" class="custom"></ion-searchbar>
   <!-- <input type="text" class="custom-search" placeholder="Търси събитие"> -->
   <div class="select-section">
-    <app-select [placeholder]="Category" class="select-item"></app-select>
-    <app-select [placeholder]="Location" class="select-item"></app-select>
+    <ion-list class="select">
```

Comment:
why not in component instead of twice here?

Reply:
It was in a component but I had a problem passing the data, from grandchild to parent so I had to put it in the child component.

Reply:
I tried repeatedly, but it didn't work.